src/org/pahappa/systems/ticketing/models/Ticket.java

```diff
@@ -1,11 +1,34 @@
 package org.pahappa.systems.ticketing.models;
 
+import org.pahappa.systems.ticketing.constants.TicketStatus;
+
 /**
  * A ticket refers to a unit of work or a request that is submitted
- * by a user or customer to seek assistance, report an issue, or request a service.
- * It serves as a record or a container that captures all the relevant information
+ * by a user or customer to seek assistance, report an issue, or request a
+ * service.
+ * It serves as a record or a container that captures all the relevant
+ * information
  * related to the user's request or issue.
  */
 public class Ticket {
+    public String Client;
+    public String Contact;
+    public String Category;
+    public String Description;
+    public TicketStatus status;
+    public String PriorityLevel;
+
+    public Ticket() {
```

Comment:
Use this resource to understand encapsulation https://beginnersbook.com/2013/05/encapsulation-in-java/ then update your attributes accessibility accordingly.